doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+reconstructed product is, in fact, *close* to our original matrix (that is, the
+difference between the two arrays is small):
+
+::
+
+    >>> np.allclose(blue_array, U_blue @ Sigma_blue @ Vt_blue)
+    True
+
+To see if an approximation is reasonable, we can check the values in ``s_blue``:
+
+::
+
+    >>> import matplotlib.pyplot as plt
+    >>> plt.plot(s_blue)
+
+In the graph, we can see that although we have 768 singular values in
+``s_blue``, most of those (after the 150th entry or so) are pretty small. So it
+might make sense to use only the information related to the first (say, 50)
+*singular values* to build a more economic approximation to our image.
+
+The idea is to consider all but the first ``k`` singular values in
+``Sigma_blue`` (which are the same as in ``s_blue``) as zeros, keeping
+``U_blue`` and ``Vt_blue`` intact, and computing the product of these matrices
+as the approximation.
+
+For example, if we choose 
+
+::
+
+    >>> k = 10
+
+we can build the approximation by doing
+
+::
+
+    >>> approx_blue = U_blue @ Sigma_blue[:, :k] @ Vt_blue[:k,:]
+
+Note that we had to use only the first ``k`` rows of ``Vt_blue``, since all
+other rows would be multiplied by the zeros corresponding to the singular
+values we eliminated from this approximation.
+
+To see if this approximation makes sense in our image, we must use a colormap
+from `matplotlib` corresponding to the color we wish to see in each
+individual image (otherwise, `matplotlib` will default to a colormap that does
+not correspond to the real data).
+
+In our case, we are approximating the blue portion of the image, so we will use
+the colormap ``Blues``:
+
+::
+
+    >>> plt.imshow(approx_blue, cmap=plt.cm.Blues)
+
+Now, you can go ahead and repeat this experiment with other values of `k`, and
+each of those should give you a slightly better (or worse) image depending on
+the value you choose.
+
+Just for good measure, to see how the original blue array looks like, you can
+do
+
+::
+
+    >>> plt.imshow(blue_array, cmap=plt.cm.Blues)
+
+Applying to all colors
+----------------------
+
+Now, we want to do the same kind of operation, but to all three colors. Our
+first instinct might be to repeat the same operation we did above to each color
+matrix individually. However, numpy's `broadcasting` takes care of this
+for us.
+
+If our array has more than two dimensions, then the SVD can be applied in
+“stacked” mode. However, the linear algebra functions in numpy expect to see a
+stacking of the form ``(N, M, M)``, where the first axis represents the number
+of stacked matrices (i.e. ``(N, M, M)`` is interpreted as a “stack” of ``N``
+matrices, each of size ``(M, M)``.)
+
+In our case,
+
+::
+
+    >>> img_array.shape
+    (768, 1024, 3)
+
+so we need to permutate the axis on this array to make get a shape like
+``(3, 768, 1024)``. Fortunately, the `numpy.transpose` function can do that for
+us:
+
+``np.transpose(x, axes=(i, j, k))`` 
```

Comment:
Should this be a code block?

Also, do we want to show `np.moveaxis(x, -1, 0)` here instead?

Reply:
+1 for changing to a code block. no need for `moveaxis` I think, that's a personal preference thing